src/management-system-v2/app/(dashboard)/iam/roles/page.tsx

```diff
@@ -1,11 +1,27 @@
-import Auth from '@/components/auth';
+import Auth, { getCurrentUser } from '@/components/auth';
+import Content from '@/components/content';
+import { getRoles } from '@/lib/data/legacy/iam/roles';
 import RolesPage from './role-page';
 
+const Page = async () => {
+  const { ability } = await getCurrentUser();
+
+  const roles = getRoles(ability);
+
+  return (
+    <Content title="Identity and Access Management">
+      <RolesPage roles={roles} />
+    </Content>
+  );
+};
+
 export default Auth(
   {
     action: 'manage',
     resource: 'Role',
     fallbackRedirect: '/',
   },
-  RolesPage,
+  Page,
 );
+
+export const dynamic = 'force-dynamic';
```

Comment:
Not necessary as long as `getCurrentUser()` is called.

Reply:
Fixed in one of my last commits